cc/toolchains/llvm/x86_64-aarch64-linux/BUILD.bazel

```diff
@@ -115,7 +115,7 @@ cc_toolchain_config(
         "//conditions:default": [],
     }),
     host_system_name = X86_64_LINUX,
-    target_cpu = "k8",
+    target_cpu = "k8",  # this should be aarch64?
```

Comment:
> this should be aarch64?
Would `k8` be a placeholder for kubernetes often abbreviated to k8s? This is indeed strange